packages/editor/src/components/commands/index.js

```diff
@@ -247,6 +248,46 @@ const getEditorCommandLoader = () =>
 			},
 		} );
 
+		commands.push( {
```

Comment:
Should this be in the editor package instead of another one more related?

Also, I'm not experienced with the command palette, but making REST API calls for commands feels like it would be pretty common. If so, this `callback` is likely something that should be (or may already be) abstracted to avoid repeating code like setting headers, making the request, processing the response, etc.